app/packs/src/apps/omniauthCredential/OmniauthCredential.js

```diff
@@ -1,48 +1,49 @@
-/* eslint-disable react/forbid-prop-types */
-import React, { useState } from 'react';
-import { Row, Col } from 'react-bootstrap';
+import React, { useState, useEffect } from 'react';
+import { Card, Row, Col } from 'react-bootstrap';
 import uuid from 'uuid';
 import UsersFetcher from 'src/fetchers/UsersFetcher';
 import NotificationActions from 'src/stores/alt/actions/NotificationActions';
-import Panel from 'src/components/legacyBootstrap/Panel'
 
-function OmniauthCredential() {
+const OmniauthCredential = () => {
   const notify = (_params) => {
     NotificationActions.add({
-      title: _params.title, message: _params.msg, level: _params.lvl, position: 'tc', dismissible: 'button', uid: uuid.v4()
+      title: _params.title,
+      message: _params.msg,
+      level: _params.lvl,
+      position: 'tc',
+      dismissible: 'button',
+      uid: uuid.v4()
     });
   };
 
-  const [providers, setProviders] = useState(() => {
-    UsersFetcher.fetchUserOmniauthProviders().then((json) => {
+  const [providers, setProviders] = useState({});
+  useEffect(async () => {
+    try {
+      const json = await UsersFetcher.fetchUserOmniauthProviders();
       const userProviders = (json?.current_user?.providers) || {};
-
       setProviders(userProviders);
-    }).catch((err) => notify({ title: 'Fetch User Providers Error', lvl: 'error', msg: err }));
-    return (providers);
-  });
-
-  let omniauthProviders = (<span />);
+    } catch (err) {
+      notify({ title: 'Fetch User Providers Error', lvl: 'error', msg: err });
+    }
+  }, []);
 
-  if (typeof omniauthProviders !== 'undefined' && omniauthProviders !== null) {
-    omniauthProviders = providers && Object.keys(providers).map((key) => (
-      <Row key={uuid.v1()}>
-        <Col key={uuid.v1()} md={2} />
-        <Col key={uuid.v1()} md={5}>{key}</Col>
-        <Col key={uuid.v1()} md={5}>{providers[key]}</Col>
-      </Row>
-    ));
-  }
+  const omniauthProviders = Object.keys(providers).map((key) => (
+    <Row key={key} className="mb-3">
+      <Col md={5}>{key}</Col>
+      <Col md={5}>{providers[key]}</Col>
```

Comment:
always use the xs parameter in place of md. See migration handbook best practices.